lib/Chat/Parser/UserMention.php

```diff
@@ -138,6 +138,7 @@ public function parseMessage(Message $chatMessage): void {
 					'id' => $chatMessage->getRoom()->getToken(),
 					'name' => $chatMessage->getRoom()->getDisplayName($userId),
 					'call-type' => $this->getRoomType($chatMessage->getRoom()),
+					'icon-url' => $chatMessage->getRoom()->getAvatar(),
```

Comment:
you need to call `AvatarService::getAvatarUrl()` ? Otherwise you just expose the "version" not a ready-to-use URL

Reply:
Good observation. Fixing and checking if is possible cover this with tests